recipes/pyxpcm/LICENSE.txt

```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```

Comment:
This file looks like it's included on the GitHub tag?

Reply:
Most likely but it was left behind when I switched. I'll fix it in the next iteration.